goosebit/settings/schema.py

```diff
@@ -61,9 +67,23 @@ def settings_customise_sources(
         dotenv_settings: PydanticBaseSettingsSource,
         file_secret_settings: PydanticBaseSettingsSource,
     ) -> tuple[PydanticBaseSettingsSource, ...]:
-        return (
+        settings_sources = [
             init_settings,
```

Comment:
Do you know where `init_settings` comes from? As it is at the top, I should have the highest priority. Not sure if that is what we want.

Reply:
I think you have it in the wrong direction still?  Init settings should be default settings I assume.

Reply:
Notice how `env_settings`  comes after the yaml file handler, as ENV always has precedence over the yaml files.  Same thing here, because `init_settings` is first in the tuple, it is the first to get parsed and can then be overwritten.

Reply:
I am a little confused because the docs state

> The order of the returned callables decides the priority of inputs; first item is the highest priority.

https://docs.pydantic.dev/latest/concepts/pydantic_settings/#customise-settings-sources

Reply:
> Do you know where init_settings comes from? As it is at the top, it should have the highest priority. Not sure if that is what we want.

Seems like these are settings passed to `__init__`, so can remove that as well.